
Traceback (most recent call last):
  File "tests/exceptions/source/others/catch_as_decorator_without_parentheses.py", line 14, in <module>
    c(2)
  File "tests/exceptions/source/others/catch_as_decorator_without_parentheses.py", line 11, in c
    a / b
ZeroDivisionError: division by zero
